product_development.md

```diff
@@ -0,0 +1,118 @@
+
+Engineers are responsible for maintaining cards on the Kanban board once they have pulled them into the backlog. Stories are always assigned when they are in progress, but never while they are in the backlog.
+
+All work that is done in either product, design or engineering should be reflected on the board. Stakeholders may have tickets on the board too.
+
+### OKR initiatives
+
+Strategic initiatives developed with a department and prioritized in close coordination with an objective owner. Those initiatives could be features and also research tasks, surveys, content work and more.
+
+They are prioritized over anything else to ensure the success of our OKRs.
+
+OKR initiatives should be in line with our long-term product vision.
+
+OKR initiatives are usually tracking in a separate lane on the Kanban board.
+
+### New feature tickets
+
+New features are the result of analysis or research based user demand.
+The product and design team owns the requirements engineering and conception of new features.
+They are responsible for creating tickets that are ready to be pulled into the backlog.
+
+### Growth opportunity
+
+Anything that can help a department (not only product related) to improve their performance and the performance of our business.
+They should be accompanied by metrics, their current state, and the projected state after the implementation of the growth opportunity.
+
+E.g: Feature A will boost KPI B from 10% to 20% and will increase revenue by 10%.
+
+Growth opportunities should align with our long term product vision.
+
+Depending on the complexity those opportunities might be developed in small increments prioritizing value delivery.
+
+Growth opportunity are owned by a stakeholder and prioritized via the stakeholder meeting.
+The product team is responsible to ensure that the ticket is ready to be pulled into the backlog.
+
+### Operational hiccups
+
+Operational hiccups are issues that disturb a stakeholders operation or hinder their ability to exercise their responsibilities.
+They usually require some kind of technical support or facilitation and should have some level of urgency similar to bugs.
+
+Operational hiccups are owned by a stakeholder and prioritized via the stakeholder meeting.
+The product team is responsible to ensure that the ticket is ready to be pulled into the backlog.
+
+### Bugs
+
+Bugs are owned by the engineering team as they affect the platform's stability.
+
+Bugs must have some kind of urgency. They are not prioritized by product, but triaged by engineering, based on what bug will kill our business first. Triage is handled by a designated senior engineer. Bugs are assigned to an engineer at all times but can be reassigned.
+
+A bug ticket should have an average resolution rate of less than a week and no more than two weeks.
+
+Bugs may be closed or converted into a technical dept ticket, if there is no longer a high urgency and no impact on the stability of the platform.
```

Comment:
Bugs may be closed or converted into a technical _**debt**_ ticket

Reply:
```suggestion
Bugs may be closed or converted into a technical debt ticket, if there is no longer a high urgency and no impact on the stability of the platform.
```